data/map.csv

```diff
@@ -0,0 +1,7 @@
+"row","issue"
+1,"mrc-768"
+2,"mrc-769"
+3,"mrc-770"
+4,"mrc-771"
+5,"mrc-772"
+6,"mrc-773"
```

Comment:
Is we make a change to this repo, kill the service, pull the repo and restart it will it count any issues after no 6 again? Or is pushing from the server the updates to `map.csv` part of the workflow of updating?

Reply:
We'll need to arrange to update the file from the server up to the repo at some point, yes. We have a facility to that with a github robot user, but it's not here yet. Pulling is safe though, because either upstream hasn't changed this file leaving the server's dirty version, or it has, in which case the pull will fail

Reply:
More generally, this is a _terrible_ place to put the global state, but it's not totally obvious how to add it to the YT tickets in a way that we can get back out, so the current situation is definitely a bodge job